nx.json

```diff
@@ -31,15 +31,19 @@
   ],
   "release": {
     "projectsRelationship": "independent",
-    "git": {
-      "commit": false
-    },
     "version": {
       "generatorOptions": {
-        "packageRoot": "{projectRoot}/dist"
+        "packageRoot": "{projectRoot}"
       },
       "conventionalCommits": true
     },
+    "changelog": {
+      "projectChangelogs": {
+        "renderOptions": {
+          "authors": false
```

Comment:
Otherwise it will add 
```
### ❤️  Thank You

- Noel Chou
````
to the changelog. I figured we don't need this since authors are on the linked commit anyway. But I guess if we anticipate having many random contributors we might want it?